Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL := /bin/bash
+
+.PHONY: tools lint build dependencies test
+
+tools:
+	# install gometalinter
+	go get -u github.com/alecthomas/gometalinter
+
+	# install gox
+	go get -v github.com/mitchellh/gox
+
+	# install all known linters:
+	gometalinter --install
+
+lint:
+	gometalinter --config="linters.json" ./...
+
+build:
+	@mkdir -p bin/
+	go build -o ./bin/vault-init
+	@echo
+
+	@./bin/vault-init version
+dependencies:
+
```

Comment:
I might have fudged up on the suggestion here (GitHub UI is a bit weird on multi-line suggestions. Could you review the spacing in the entire file.

Reply:
Cool. So we'll remove the extra spaces that were added after the commands?

Reply:
Yes ... like `tools` and `lint` 👇 

```makefile
SHELL := /bin/bash

.PHONY: tools lint build dependencies test

tools:
	# install gometalinter
	go get -u github.com/alecthomas/gometalinter

	# install gox
	go get -v github.com/mitchellh/gox

	# install all known linters:
	gometalinter --install

lint:
	gometalinter --config="linters.json" ./...

build:
	@mkdir -p bin/
	go build -o ./bin/vault-init
	@echo

	@./bin/vault-init version
dependencies:

	dep ensure
test:
	go test -v ./...
```

Reply:
This hasn't been addressed (also edited previous comment who's example was wrong ... forgot to edit after copy).